tfhe/src/integer/client_key/radix.rs

```diff
@@ -131,6 +142,40 @@ impl RadixClientKey {
     pub fn num_blocks(&self) -> usize {
         self.num_blocks
     }
+
+    pub fn new_compression_private_key(
```

Comment:
I'm not sure we want to add things to the client key in this PR :thinking: @mayeul-zama or @tmontaigu maybe you could give us some advice here.

Reply:
I think you can use this: tfhe/src/shortint/list_compression/compressed_server_keys.rs:76

Reply:
When I replace the client key by a shortint client key the compression test won't pass anymore. I'm clearly making something wrong. @tmontaigu can you help?

Reply:
@IceTDrinker maybe?

Reply:
sorry a bit hard to know what is needed here ?